sklearn/plot/_confusion_matrix.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_confusion_matrix(values, classes, normalize=False,
```

Comment:
then we'll have to also give a "sample_weights" optional argument as in confusion matrix? also, should be give an optional argument to return the confusion matrix as an array? with these, they won't need to call the confusion_matrix function. thoughts?